serverless.yml

```diff
@@ -38,6 +41,18 @@ custom:
       Type: String
       Default: ${env:GITHUB_BUILD_COMMENT, self:custom.github-webhook.githubBuildComment}
       Description: For pull request comments, the comment that will initiate a build
+    CbExternalBuildspec:
+      Type: String
+      Default: ${env:CB_EXTERNAL_BUILDSPEC, self:custom.github-webhook.cbExternalBuildspec}
+      Description: If true, CodeBuild will not send "sourceVersion" with request for buildspec (e.g., allowing buildspec to be in an external repo not on same PR number)
```

Comment:
"Controls whether the sourceVersion parameter is set to the pull request and passed to the CodeBuild job. Set to true if the repo for the CodeBuild project itself is different from the repo where the webhook will be created"